test/puppetlabs/puppetdb/http/nodes_test.clj

```diff
@@ -567,7 +567,11 @@
 
       (is-query-result' ["=" "report_timestamp" web1-report-ts] [web1])
       (is-query-result' [">" "report_timestamp" web1-report-ts] [db puppet])
-      (is-query-result' [">=" "report_timestamp" web1-report-ts] [web1 db puppet]))))
+      (is-query-result' [">=" "report_timestamp" web1-report-ts] [web1 db puppet])))
+
+  (let [{:keys [body status]} (query-response method endpoint ["<" "catalog_timestamp" "'2018-08-15 21:11:21 UTC'"])]
```

Comment:
PR looks good, though this line's getting a bit long -- suggest maybe one of these, or some other variation/accommodation:
```
  (let [{:keys [body status]} (->> ["<" "catalog_timestamp" "'2018-08-15 21:11:21 UTC'"]
                                   (query-response method endpoint))]
    ...)
  
  (let [{:keys [body status]}
        (query-response method endpoint ["<" "catalog_timestamp" "'2018-08-15 21:11:21 UTC'"])]
    ...)

  ;; (Also plausible to not use :keys here, since we only refer to each field once....)
  (let [res (->> ["<" "catalog_timestamp" "'2018-08-15 21:11:21 UTC'"]
                 (query-response method endpoint))]
    (is (= 400 (:status res))))
```